src/gennylib/src/context.cpp

```diff
@@ -96,6 +101,18 @@ ActorVector WorkloadContext::_constructActors(const Cast& cast,
     return actors;
 }
 
+mongocxx::pool::entry WorkloadContext::client() {
+    if (_hasApmOpts) {
+        // TODO: Remove this conditional when CDRIVER-2931 is resolved.
```

Comment:
in this and other places related to the APM callback let's add a `// TODO(TIG-XYZ) ...`